docs/HACKING.md

```diff
@@ -291,3 +291,41 @@ the `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE` environment variable like so:
 ```
 OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=quay.io/user/origin-release:v{version number} bin/openshift-install create cluster --log-level=debug
 ```
+
+## Hacking on `machine-os-content`
+
+If you own part of the operating system (from kernel to kubelet) you
+are part of the `machine-os-content`.  More information in [OSUpgrades.md].
+You will want a workflow for testing changes to a cluster.
+
+By far the simplest in a lot of cases is going to be to use `oc debug node/`
+to start by testing your changes on e.g. a single worker node.
```

Comment:
"The simplest workflow is to use `oc debug node/` to start testing you  changes on a single worker node"

Reply:
"This is a good method to quickly test your changes, but you'll want to also choose method #3 to deploy to a cluster" <-- or something?